src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizerConfigurationValidator.cs

```diff
@@ -62,6 +61,11 @@ public void Validate(AnonymizerConfiguration config)
                 {
                     throw new AnonymizerConfigurationErrorsException($"Missing replaceWith value in substitution rule at {rule[Constants.PathKey]}.");
                 }
```

Comment:
If we put the validation of perturb in PerturbSetting, would it be better to keep the same style for substitution?

Reply:
Yes, I have moved the rule setting validation to SubstituteSetting